src/amp.js

```diff
@@ -39,6 +40,9 @@ import {maybeValidate} from './validator-integration';
 try {
   // Should happen first.
   installErrorReporting(window);  // Also calls makeBodyVisible on errors.
+  let perf = performanceFor(window);
+
+  perf.tick('installStyles');
```

Comment:
Its pretty common to give these super short names like "is". Maybe create an .md file with a table?